horovod/tensorflow/mpi_ops.py

```diff
@@ -201,7 +201,7 @@ def _broadcast_grad(op, grad):
     return grad_reduced
 
 
-def _reducescatter(tensor, name=None):
+def _reducescatter(tensor, name=None, op=Sum):
```

Comment:
Looks like everywhere else the default is `Average` but here it's `Sum`.  Is there a reason, or should we make it `Average` here as well?

Reply:
So, `Average` needs to be handled in the higher-level `reducescatter()` in `__init__.py` (which is where the division happens).  I was trying to match the behavior of Allreduce, where `allreduce()` handles `Average` and passes `Sum` to `_allreduce()`.

`_allreduce()` and `_reducescatter()` defaults are `Sum`
`allreduce()` and `reducescatter()` defaults are `Average`

Reply:
Let me make a small tweak to `reducescatter()` though.  Since the underlying `_reducescatter()` only supports `Sum` at the moment, I never actually pass the reduce op through to it.  However, I should pass something to it since any errors will get caught in `EnqueueTensorReduceScatter` anyway so it's "safe" to pass the op.  That way future people aren't confused why they pass `Max` and it doesn't show up in the C++ API.